### ISSUE TITLE
error report and fix bug with snap package in ubuntu Ubuntu 18.04 bionic

### ISSUE THREAD

**Issue by itonedeveloper** [Almighty-Alpaca/JetBrains-Discord-Integration#94]
hello i have phpstorm, 
im installing discord from snap package manager 
but i have error in Discord-Integration

```
java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.collections.ArraysKt___ArraysKt.contains, parameter $this$contains
	at kotlin.collections.ArraysKt___ArraysKt.contains(_Arrays.kt)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$readPlugins$matches$4.invoke(DiagnoseService.kt:153)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$readPlugins$matches$4.invoke(DiagnoseService.kt:40)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseServiceKt$sam$java_util_function_Predicate$0.test(DiagnoseService.kt)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:257)
	at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:248)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.count(ReferencePipeline.java:605)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService.readPlugins(DiagnoseService.kt:154)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService.access$readPlugins(DiagnoseService.kt:40)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$plugins$1.invokeSuspend(DiagnoseService.kt:44)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**it seems like discord is running in a snap package this will most likely prevent the plugin from connecting to your discord client**

![phpstorm](https://user-images.githubusercontent.com/39291525/81466497-2896fc80-91a0-11ea-85b0-5d1821251089.png)

I  think the Discord Integration plugin will does not connect to  discord from snap package

**Comment by duncte123** [comment]
Don't use the snap package as that version is outdated and does not support RPC.
The exception seems to be a duplicate of #92

**Comment by InfRandomness** [comment]
Hello, Snap is designed in such a way that it runs the installed applications in a sandbox, which means that the external applications cannot communicate with the discord client's RPC system. I recommend you to [download](https://discord.com/) and install Discord using the official releases.

**Comment by itonedeveloper** [comment]
thanks @duncte123  @InfRandomness

**Comment by InfRandomness** [comment]
A fix for the exception has been released 12h ago on the repository.

**Comment by Almighty-Alpaca** [comment]
A fix for this was released this morning with version v1.4.0
